pkgs/applications/misc/spacefm/x11-only.patch

```diff
@@ -0,0 +1,10 @@
+--- a/src/main.c	2021-02-09 13:54:32.847364236 +0100
++++ b/src/main.c	2021-02-09 10:41:51.541203271 +0100
+@@ -1350,6 +1351,7 @@
+                 vfs_file_monitor_clean();
+                 return 1;
+             }
++            gdk_set_allowed_backends("x11");
```

Comment:
In case this project is still maintained it might be useful to send this upstream independently of this pr.

Reply:
There has been no activity in this project for years, with still a number of open PRs. I guess that would only be a waste of time.